packages/components/src/interfaces/complex/InfoCards.ts

```diff
@@ -49,6 +49,9 @@ const InfoCardsBaseSchema = Type.Object({
       default: "This is an optional description for the Cards component",
     }),
   ),
+  maxColumns: Type.Optional(
+    Type.Union([Type.Literal(1), Type.Literal(2), Type.Literal(3)]),
+  ),
```

Comment:
issue: this needs a title and description so it can render the input field correctly on Studio.

Reply:
Hmm was planning for this to be a hidden config and not changeable (since it’s set by the renderer), but I guess I’ll just add it in first